patch/series

```diff
@@ -107,6 +107,9 @@ cisco-npu-disable-other-bars.patch
 #
 # Marvell platform patches for 4.19
 armhf_secondary_boot_online.patch
+
+# Micas patches
```

Comment:
Explain, what Micas is?

Reply:
sorry，have modified  ”Micas“ to “optoe“，please help to review again ,thanks you !